src/Testing/InteractsWithMail.php

```diff
@@ -147,4 +147,9 @@ protected function seeMessage()
 
         return $this;
     }
+
+    protected function dontSeeMessage()
+    {
+        $this->assertTrue($this->getMessages()->isEmpty());
```

Comment:
Why not make this method the inverse of "seeMessage"?
- assertNull
- add helpful assertion message
- make fluent by adding `return $this;`